decompose-router/src/androidInstrumentedTest/kotlin/io/github/xxfast/decompose/screen/TestScreens.kt

```diff
@@ -56,12 +59,17 @@ const val TITLEBAR_TAG = "titleBar"
 const val DETAILS_TAG = "details"
 const val LAZY_COLUMN_TAG = "lazyColumn"
 
+@OptIn(ExperimentalDecomposeApi::class)
 @Composable
 fun HomeScreen() {
   val router: Router<Screen> = rememberRouter(Screen::class, listOf(Screen.Game))
   RoutedContent(
     router = router,
-    animation = stackAnimation(slide())
+    animation = predictiveBackAnimation(
+      animation = stackAnimation(slide()),
+      onBack = { router.pop() },
+      backHandler = LocalComponentContext.current.backHandler
```

Comment:
Hi @arkivanov, I'm using the backhander from the root component context here. Would this be an issue?

Reply:
Possibly yes, it's better to grab the back handler from LocalComponentContext, of expose it from the Router. Also mind this big: https://github.com/arkivanov/Decompose/issues/403